src/audio/env.c

```diff
@@ -482,7 +482,9 @@ static s16 _getRate(f64 vol, f64 tgt, s32 count, u16* ratel) {
         }
     }
 
-    a *= (a *= (a *= a));
+    a *= a;
+    a *= a;
+    a *= a;
```

Comment:
This feels unbelievably cursed, can we just do something like `a = a * a * a;` instead? Maybe throw in a `sqr` macro if there is one (and create one somewhere if there isn't)?

Reply:
This is a^8, this is easiest to see in SSA form:
b = a * a;
c = b * b;
d = c * c;

Removing the intermediate variables results in:
d = a * a * a * a * a * a * a * a;

Reply:
I even double checked myself to make sure I wasn't an idiot and still failed...